docker-compose.yml

```diff
@@ -1,8 +1,8 @@
 version: "2"
 services:
   server:
-    image: gcr.io/jenkinsxio/hugo-extended:0.65.3-6
-    command: ["hugo", "server", "-s", "/src", "-D", "--bind", "0.0.0.0", "--ignoreCache"]
+    image: klakegg/hugo:0.73.0-ext-alpine
```

Comment:
This is just for preview/demo, need to discuss before removing the hugo extended base image